### PR TITLE
[ES] Native review of `History of osu!/2013`

### PR DESCRIPTION
**Pull request by UnknownOsu**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

**Review by JuanM4T — APPROVED** [review]

File: wiki/History_of_osu!/2013/es.md [hunk 1]
```diff
@@ -1,130 +1,126 @@
...
+Para el modo multijugador, los jugadores ahora pueden usar el mod especial (Relax/Autopilot/SpunOut/ManiaKeys) cuando el anfitrión habilite el Mod libre. En el lado de skinning, se han agregado sprites predeterminados de pantalla ancha de taiko (taiko `@2x`). Los favoritos A/B se eliminaron por completo y fueron reemplazados por las colecciones.
 
-Se han lanzado tres nuevos [logros](/wiki/Medals) de dedicación para osu!mania (40 000 / 400 000 / 4 000 000 teclas presionadas). El diseño del perfil de usuario se modificó y su "última visita" se basará en la conexión a Bancho (es decir, el inicio de sesión en el juego) en lugar de las visitas al foro. ¡Finalmente, osu! golpea el hito de tener 10,000 usuarios conectados simultáneamente!
+Se han lanzado tres nuevos [logros](/wiki/Medals) de dedicación para osu!mania (40 000 / 400 000 / 4 000 000 teclas presionadas). El diseño del perfil de usuario se modificó y su «última visita» se basará en la conexión a Bancho (es decir, el inicio de sesión en el juego) en lugar de las visitas al foro. ¡Finalmente, osu! golpea el hito de tener 10 000 usuarios conectados simultáneamente!
```
Comment (agatemosu) [inline wiki/History_of_osu!/2013/es.md:78]:
```suggestion
Se han lanzado tres nuevos [logros](/wiki/Medals) de dedicación para osu!mania (40 000 / 400 000 / 4 000 000 teclas presionadas). El diseño del perfil de usuario se modificó y «última visita» se basará en la conexión a Bancho (es decir, el inicio de sesión en el juego) en lugar de las visitas al foro. ¡Finalmente, osu! golpea el hito de tener 10 000 usuarios conectados simultáneamente!
```

File: wiki/History_of_osu!/2013/es.md [hunk 2]
```diff
@@ -1,130 +1,126 @@
...
 
-- [Lanzamiento público de osu! (b20130509)](https://osu.ppy.sh/community/forums/topics/131611)
-- [Aspectos destacados de mayo de 2013 + Mapa del mes](https://osu.ppy.sh/community/forums/posts/2334705)
+- [osu! Public Release (b20130509)](https://osu.ppy.sh/community/forums/topics/131611)
+- [May 2013 Highlights + Map of the Month](https://osu.ppy.sh/community/forums/posts/2334705)
 
 ## Junio
 
-Ha comenzado [el concurso de mapeo de osu! \#4](https://osu.ppy.sh/community/forums/posts/2324098). Como contramedida contra los jugadores que crean muchas cuentas, se mejoró la claridad de los mensajes durante el registro de la cuenta y se agregó una infografía que muestra que no puede registrarse desde teléfonos/tabletas para evitar que los jugadores creen más cuentas por motivos personales.
+Ha comenzado [el osu! Beatmapping Contest #4](https://osu.ppy.sh/community/forums/posts/2324098). Como contramedida contra los jugadores que crean muchas cuentas, se mejoró la claridad de los mensajes durante el registro de la cuenta y se agregó una infografía que muestra que no pueden registrarse desde teléfonos/tabletas para evitar que los jugadores creen más cuentas por motivos personales.
```
Comment (agatemosu) [inline wiki/History_of_osu!/2013/es.md:85]:
```suggestion
Ha comenzado [el osu! Beatmapping Contest \#4](https://osu.ppy.sh/community/forums/posts/2324098). Como contramedida contra los jugadores que crean muchas cuentas, se mejoró la claridad de los mensajes durante el registro de la cuenta y se agregó una infografía que muestra que no pueden registrarse desde teléfonos/tabletas para evitar que los jugadores creen más cuentas por motivos personales.
```

File: wiki/History_of_osu!/2013/es.md [hunk 3]
```diff
@@ -134,9 +130,9 @@ Links:
 
 Uno de los temas candentes de debate en este momento era que peppy estaba creando un nuevo modo de juego y lo estaba probando como una prueba de concepto (probando si es jugable, competitivo o lo suficientemente flexible). Algunos jugadores especulan que fue [Touhosu!](https://osu.ppy.sh/community/forums/topics/19307) (esquivando el infierno de balas, es decir, lo contrario a [Catch the Beat](/wiki/Game_mode/osu!catch)). Al momento de escribir, no se descubrió mucha información sobre el nuevo modo de juego. [Un videoclip sobre la charla estaba en el blog de ppy.](https://blog.ppy.sh/post/71405880656/i-have-a-few-posts-waiting-for-some-final-touches)
```
Comment (agatemosu) [inline wiki/History_of_osu!/2013/es.md:131]:
```suggestion
Uno de los temas candentes de debate en este momento era que peppy estaba creando un nuevo modo de juego y lo estaba probando como una prueba de concepto (probando si es jugable, competitivo o lo suficientemente flexible). Algunos jugadores especulan que fue [Touhosu!](https://osu.ppy.sh/community/forums/topics/19307) (esquivando el infierno de balas, es decir, lo contrario a [Catch the Beat](/wiki/Game_mode/osu!catch)). Al momento de escribir, no se descubrió mucha información sobre el nuevo modo de juego. [Un videoclip sobre la charla estaba en el blog de ppy](https://blog.ppy.sh/post/71405880656/i-have-a-few-posts-waiting-for-some-final-touches).
```

File: wiki/History_of_osu!/2013/es.md
Comment (agatemosu) [inline wiki/History_of_osu!/2013/es.md:1]: peppy fue confirmado para... -> se confirmó que peppy estaría en...

File: wiki/History_of_osu!/2013/es.md [hunk 4]
```diff
@@ -1,130 +1,126 @@
...
+El 2 de julio de 2013, peppy abrió al público la [osu!api](https://osu.ppy.sh/community/forums/topics/141240) (anteriormente disponible solo a través de solicitudes privadas). Esto permitió a los desarrolladores independientes trabajar en servicios de terceros para osu!. El 11 de julio de 2013, se arregló la página «Olvidé mi contraseña» que no funcionaba anteriormente. El 18 de julio de 2013 trajo actualizaciones a la conectividad alternativa de Bancho. El 21 de julio de 2013, se agregó el soporte de pantalla ancha para storyboards en el cuadro de diálogo de configuración de la canción. Se agregaron dos nuevas superposiciones en el juego para rastrear la precisión mientras juegas o ves jugar a otros. Consulta la página [opciones](/wiki/Client/Options) para obtener más información.
```
Comment (agatemosu) [inline wiki/History_of_osu!/2013/es.md:114]:
```suggestion
El 2 de julio de 2013, peppy abrió al público la [osu!api](https://osu.ppy.sh/community/forums/topics/141240) (anteriormente disponible solo a través de solicitudes privadas). Esto permitió a los desarrolladores independientes trabajar en servicios de terceros para osu!. El 11 de julio de 2013, se arregló la página «Olvidé mi contraseña» que no funcionaba anteriormente. El 18 de julio de 2013 trajo actualizaciones a la conectividad alternativa de Bancho. El 21 de julio de 2013, se agregó el soporte de pantalla ancha para storyboards en el cuadro de diálogo de configuración de la canción. Se agregaron dos nuevas superposiciones en el juego para rastrear la precisión mientras juegas o ves jugar a otros. Consulta la página [Opciones](/wiki/Client/Options) para obtener más información.
```

File: wiki/History_of_osu!/2013/es.md [hunk 5]
```diff
@@ -1,142 +1,138 @@
...
 
-Los modos de juego ahora se pueden cambiar fácilmente en la selección de canciones usando `Ctrl` + `1`, `2`, `3` o `4`. Se agregó un nuevo modificador del juego especialmente para osu!mania, [Random](/wiki/Gameplay/Game_modifier/Random). Se han realizado algunos ajustes en el comportamiento del botón "Partida aleatoria" de Multijugador para dirigirlo a una sala con respecto a su pp promedio frente a la desviación del pp de la sala.
+Los modos de juego ahora se pueden cambiar fácilmente en la selección de canciones usando `Ctrl` + `1`, `2`, `3` o `4`. Se agregó un nuevo modificador del juego especialmente para osu!mania, [Random](/wiki/Gameplay/Game_modifier/Random). Se han realizado algunos ajustes en el comportamiento del botón «Partida aleatoria» de multijugador para dirigirte a una sala con respecto a tu pp promedio frente a la desviación del pp de la sala.
```
Comment (agatemosu) [inline wiki/History_of_osu!/2013/es.md:54]:
```suggestion
Los modos de juego ahora se pueden cambiar fácilmente en la selección de canciones usando `Ctrl` + `1`, `2`, `3` o `4`. Se agregó un nuevo modificador del juego especialmente para osu!mania, [Random](/wiki/Gameplay/Game_modifier/Random). Se han realizado algunos ajustes en el comportamiento del botón «Partida aleatoria» del multijugador para dirigirte a una sala con respecto a tu pp promedio frente a la desviación del pp de la sala.
```

File: wiki/History_of_osu!/2013/es.md [hunk 6]
```diff
@@ -1,142 +1,138 @@
...
+Ha comenzado [el osu! Beatmapping Contest \#4](https://osu.ppy.sh/community/forums/posts/2324098). Como contramedida contra los jugadores que crean muchas cuentas, se mejoró la claridad de los mensajes durante el registro de la cuenta y se agregó una infografía que muestra que no pueden registrarse desde teléfonos/tabletas para evitar que los jugadores creen más cuentas por motivos personales.
 
-Aparte de las medidas de seguridad, los usuarios podían elegir el modo de juego predeterminado de su perfil (como [osu!mania](/wiki/Game_mode/osu!mania), [Taiko](/wiki/Game_mode/osu!taiko)) y [last.fm](https://last.fm) compatibilidad con perfiles. Los perfiles de usuario ahora tienen un rango de desempeño del país en su perfil y un nuevo gráfico de pp.
+Aparte de las medidas de seguridad, los usuarios podían elegir el modo de juego predeterminado de su perfil (como [osu!mania](/wiki/Game_mode/osu!mania), [Taiko](/wiki/Game_mode/osu!taiko)) y compatibilidad con perfiles de [last.fm](https://last.fm). Los perfiles de usuario ahora tienen un rango de desempeño del país en su perfil y un nuevo gráfico de pp.
```
Comment (agatemosu) [inline wiki/History_of_osu!/2013/es.md:87]:
```suggestion
Aparte de las medidas de seguridad, los usuarios podían elegir el modo de juego predeterminado de su perfil (como [osu!mania](/wiki/Game_mode/osu!mania), [Taiko](/wiki/Game_mode/osu!taiko)) y compatibilidad con perfiles de [last.fm](https://last.fm). Los perfiles de usuario ahora tienen una clasificación por rendimiento nacional en su perfil y un nuevo gráfico de pp.
```

File: wiki/History_of_osu!/2013/es.md [hunk 7]
```diff
@@ -1,142 +1,138 @@
...
+En cuanto a la jugabilidad, el tiempo de inicio se agregó automáticamente a cualquier beatmap que comience abruptamente para permitir suficiente tiempo para ajustar la [configuración visual](/wiki/Client/Interface/Visual_settings) y se agregó el recuento total de pulsaciones para cada tecla en la superposición de entrada. Para el almacenamiento de los resultados en línea, el mejor rango personal se mostró en las páginas de información del beatmap además del jugador \#1. Se permitió el envío de puntajes desde las tablas de mania y se almacenaron las repeticiones del top 50 en lugar del top 40 habitual. Se corrigieron los sliders reversos que no alcanzaban su combo máximo. Se agregaron comentarios de Disqus a las páginas de los beatmaps. Finalmente, se estaba realizando la prueba del prototipo del medidor de puntuación para todos los modos para osu!test.
```
Comment (agatemosu) [inline wiki/History_of_osu!/2013/es.md:91]:
```suggestion
En cuanto a la jugabilidad, el tiempo de inicio se agregó automáticamente a cualquier beatmap que comience abruptamente para permitir suficiente tiempo para ajustar la [configuración visual](/wiki/Client/Interface/Visual_settings) y se agregó el recuento total de pulsaciones para cada tecla en la superposición de entrada. Para el almacenamiento de los resultados en línea, el mejor rango personal se mostró en las páginas de información del beatmap además del jugador \#1. Se permitió el envío de puntajes desde las tablas de mania y se almacenaron las repeticiones del top 50 en lugar del top 40 habitual. Se corrigieron los sliders inversos que no alcanzaban su combo máximo. Se agregaron comentarios de Disqus a las páginas de los beatmaps. Finalmente, se estaba realizando la prueba del prototipo del medidor de puntuación de todos los modos para osu!test.
```

File: wiki/History_of_osu!/2013/es.md [hunk 8]
```diff
@@ -1,142 +1,138 @@
...
 
 ## Diciembre
 
 ![](img/2013-12_01.jpg "Aparición especial en la Comic Fiesta 2013")
 
-[peppy fue confirmado para la Comic Fiesta 2013 en Malasia los días 21 y 22 de diciembre en el centro de la ciudad de Kuala Lumpur](https://osu.ppy.sh/community/forums/topics/163121). Para el día 1 (21/12/2013), osu! se mostró como un evento de escenario, demostrando el juego en cuatro modos de juego diferentes (osu!, Taiko, Catch the Beat y osu!mania). Para el día 2 (22/12/2013), peppy da una idea de osu! junto con sus futuros planes de actualización en la sala de paneles y mostró la osu!tablet para que el público la probara.
+[Se confirmó que peppy estaría en la Comic Fiesta 2013 en Malasia los días 21 y 22 de diciembre en el centro de la ciudad de Kuala Lumpur](https://osu.ppy.sh/community/forums/topics/163121). Para el día 1 (21/12/2013), osu! se mostró como un evento de escenario, demostrando el juego en cuatro modos de juego diferentes (osu!, Taiko, Catch the Beat y osu!mania). Para el día 2 (22/12/2013), peppy da una idea de osu! junto con sus futuros planes de actualización en la sala de paneles y mostró la osu!tablet para que el público la probara.
```
Comment (agatemosu) [inline wiki/History_of_osu!/2013/es.md:129]:
```suggestion
[Se confirmó que peppy estaría en la Comic Fiesta 2013 de Malasia los días 21 y 22 de diciembre en el centro de la ciudad de Kuala Lumpur](https://osu.ppy.sh/community/forums/topics/163121). Para el día 1 (21/12/2013), osu! se mostró como un evento de escenario, demostrando el juego en cuatro modos de juego diferentes (osu!, Taiko, Catch the Beat y osu!mania). Para el día 2 (22/12/2013), peppy dió una idea de osu! junto con sus futuros planes de actualizaciones en la sala de paneles y mostró la osu!tablet para que el público la probara.
```

File: wiki/History_of_osu!/2013/es.md (hunk 5 above)
Comment (UnknownOsu) [reply wiki/History_of_osu!/2013/es.md:54]: `de` creo que queda mejor en este caso

**Review by agatemosu — APPROVED** [review]

**Review by TicClick — APPROVED** [review]

**Pull request merged**